setup.cfg

```diff
@@ -1,10 +1,10 @@
 [metadata]
-name = skeleton
+name = PROJECT
 author = Jason R. Coombs
 author_email = jaraco@jaraco.com
-description = skeleton
+description = PROJECT_DESCRIPTION
```

Comment:
I'm wondering about author/author_email/description - if maybe these should be ommitted or have placeholders or what.

Reply:
I'm not going to bother with author / e-mail for now. Ideally, those values should be inferred from the git metadata, but I'm not going to solve that here.